app/lib/bulk_download_service.rb

```diff
@@ -160,21 +160,47 @@ def self.get_requested_directory_files(directories)
   # Get a preview of the number of files/total bytes by StudyAccession and file_type
   #
   # * *params*
-  #   - +file_types+ (Array<String>) => Array of requested file types to be ingested
   #   - +study_accessions+ (Array<String>) => Array of StudyAccession values from which to pull files
-  #
   # * *return*
-  #   - (Hash) => Hash of StudyFile::BULK_DOWNLOAD_TYPES matching query w/ number of files and total bytes
-  def self.get_requested_file_sizes_by_type(file_types: [], study_accessions:)
-    # replace 'Expression' with both dense & sparse matrix file types
-    requested_files = get_requested_files(file_types: file_types, study_accessions: study_accessions)
-    files_by_type = {}
-    requested_types = requested_files.map(&:simplified_file_type).uniq
-    requested_types.each do |req_type|
-      files = requested_files.select {|file| file.simplified_file_type == req_type}
-      files_by_type[req_type] = {total_files: files.size, total_bytes: files.map(&:upload_file_size).compact.reduce(:+)}
+  #   - (Hash) => Array of study objects, each one containing a study_files array with name, type, and upload_file_size
+  def self.get_download_info(study_accessions)
+    Study.where(:accession.in => study_accessions).map { |study| BulkDownloadService.study_download_info(study) }
+  end
+
+  # Get download preview information for a single study
+  #
+  # * *params*
+  #   -+study+ (Study) => The study to retrieve download preview information for
+  #
+  # * *returns*
+  #   - (Hash) => Hash of download information about the study, including an array of the study files
+  def self.study_download_info(study)
+    {
+      name: study.name,
+      accession: study.accession,
+      description: study.description,
+      study_files: study.study_files.map { |study_file| BulkDownloadService.study_file_download_info(study_file) }
```

Comment:
I had thought that we were only doing the 3 'main' types for bulk downloads.  Are we changing to "all files" by default now?  We may run into issues w/ studies containing sequence data.  Just a thought.

Reply:
Good question, Eric and I just chatted about this.  I think the right approach is to have the backend only worry about auth. and quota, and otherwise just do whatever the frontend asks.  So it will give the frontend a list of all available files, and the frontend can handle deciding what to ask for.  

 This will make it easier to update the UX to support additional ways of downloading.  I did update the actual download method so that if you just provide accessions, without supplying any file types, it defaults to only giving you the main 3.